src/OrbitGl/App.cpp

```diff
@@ -168,6 +168,8 @@ using UnwindingMethod = orbit_grpc_protos::CaptureOptions::UnwindingMethod;
 namespace {
 
 constexpr const char* kLibOrbitVulkanLayerSoFileName = "libOrbitVulkanLayer.so";
+constexpr const char* kNtdllSo = "ntdll.so";
```

Comment:
The one above has suffix `FileName`. Maybe this should too, and the one below should be `kWineSyscallDispatcherFunctionName`?

Reply:
Done